src/tests/ftest/container/attribute.py

```diff
@@ -80,6 +81,7 @@ class ContainerAttributeTest(TestWithServers):
     def setUp(self):
         super(ContainerAttributeTest, self).setUp()
 
+        self.logger = logging.getLogger("ConAttributeLogger")
```

Comment:
Any reason to use a separate logger when **apricot** already has a `DaosLog` handle open?

https://github.com/daos-stack/daos/blob/db0eec3dd6b55a86b69b5d35561f6ba9d4d646d4/src/tests/ftest/util/apricot/apricot/test.py#L129

Reply:
this apricot d_log is for TestWithoutServers class, somehow it's not defined in TestWithServers class that used here.
Two solutions: 
1. Add the d_log definition in TestWithServers
2. Define it locally in attributeTest classes

Suggestions?

Reply:
TestWithServers inherits from TestWithoutServers:
https://github.com/daos-stack/daos/blob/b7c562626c652e0b7a63c7c0f693bd020234ed0d/src/tests/ftest/util/apricot/apricot/test.py#L139